package.json

```diff
@@ -90,33 +82,28 @@
     "@types/minimatch": "^3.0.0",
     "@types/node": "^8.0.24",
     "@types/shelljs": "^0.7.4",
-    "babel-eslint": "^8.2.1",
     "del-cli": "^1.1.0",
-    "eslint": "^4.18.0",
-    "eslint-config-airbnb": "^16.1.0",
-    "eslint-config-prettier": "^2.9.0",
-    "eslint-plugin-import": "^2.8.0",
-    "eslint-plugin-jsx-a11y": "^6.0.3",
-    "eslint-plugin-react": "^7.6.1",
+    "eslint": "^4.18.2",
     "husky": "^0.15.0-rc.8",
     "lint-staged": "^7.0.0",
     "npm-run-all": "^4.0.2",
-    "prettier": "^1.11.1",
-    "rxjs": "^5.0.0",
-    "rxjs-tslint-rules": "^3.14.0",
-    "stylelint": "^8.0.0",
-    "stylelint-config-standard": "^17.0.0",
-    "stylelint-no-unsupported-browser-features": "^1.0.0",
+    "prettier": "^1.10.2",
     "tslint": "^5.9.1",
-    "tslint-config-prettier": "^1.8.0",
-    "tslint-eslint-rules": "^5.0.0",
-    "tslint-microsoft-contrib": "^5.0.3",
-    "tslint-react": "^3.5.1",
     "typescript": "^2.8.1",
-    "typescript-eslint-parser": "^14.0.0",
     "yarnhook": "^0.1.1"
   },
   "engines": {
     "node": ">= 6.10"
+  },
+  "private": false,
+  "jest": {
+    "globals": {
+      "ts-jest": {
+        "useBabelrc": true
+      }
+    }
+  },
```

Comment:
When creating the Jest and Babel extension folders in hollowverse/config, I had to decide which of these folders should have this configuration. I ended up putting it in the Babel folder since it's harmless to have it. Unfortunately, Clown doesn't provide a way to selectively delete a configuration like this. Maybe we can think of a solution for this later.

Reply:
I see. By the way, jest supports configuration defined in a separate `jest.config.js` file (or a JSON file via `jest --config </path/to/json>`).